static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -115,6 +115,18 @@
     }
 }
 
+.advert__kicker {
+    &:after {
+        content: '/';
+        display: inline-block;
+        margin-left: .2em;
+        color: mix(#ffffff, $news-support-6, 20%);
```

Comment:
Yeah, took  a bit of time to track down where the actual RGB colour on the paid-for rich links was coming from :weary: